spec/support/helpers.rb

```diff
@@ -40,6 +40,14 @@ def create_bulk_user
   user
 end
 
+def create_merger_user
+  sf_user = FactoryBot.create(:sf_guard_user)
+  user = FactoryBot.create(:user, sf_guard_user_id: sf_user.id)
+  SfGuardUserPermission.create!(permission_id: 3, user_id: sf_user.id)
```

Comment:
magic number alert! (but fine.)

Reply:
this is a good point. we actually don't even have those permissions constants defined anywhere...probably should.